src/applications/ivc-champva/10-10D/helpers/utilities.js

```diff
@@ -40,36 +41,74 @@ export const applicantListSchema = (requireds, propertyList) => {
   };
 };
 
+// Extracting this to a function so there aren't a thousand identical
+// ternaries we have to change later
+export function sponsorWording(formData, isPosessive = true, cap = true) {
+  const retVal = isPosessive ? 'sponsor’s' : 'sponsor';
+  // Optionally capitalize first letter and return
+  return cap ? retVal.charAt(0).toUpperCase() + retVal.slice(1) : retVal;
+}
+
 /**
- * Gets the appropriate form key needed in the signature box so we can
- * corroborate who is signing the form.
- *
- * @param {object} formData All data currently in the form
- * @returns
+ * Adds a new `applicant` object to the start of the `formData.applicants`
+ * array. This is used to add the certifier info to the first applicant
+ * slot so users don't have to enter info twice if the certifier is also an app.
+ * @param {object} formData standard formData object
+ * @param {string} path formconfig page path of the page where this function should apply
+ * @param {object} name standard fullNameUI name to populate
+ * @param {string} email email address to populate
+ * @param {string} phone phone number to populate
+ * @param {object} address standard addressUI address object to populate
  */
-export function getNameKeyForSignature(formData) {
-  let nameKey;
-  if (formData.certifierRole === 'sponsor') {
-    nameKey = 'veteransFullName';
-  } else if (formData.certifierRole === 'applicant') {
-    nameKey = 'applicants[0].applicantName';
+export function populateFirstApplicant(
+  formData,
+  path,
+  name,
+  email,
+  phone,
+  address,
+) {
+  // Make sure we apply this change when the user is on the certifier relationship
+  // page, as firing later in the form would potentially add duplicate info:
+  if (!window.location.href.endsWith(path)) return formData;
+
+  const modifiedFormData = formData; // changes will affect original formData
+  const newApplicant = {
+    applicantName: name,
+    applicantEmailAddress: email,
+    applicantAddress: address,
+    applicantPhone: phone,
+  };
+  if (modifiedFormData.applicants) {
+    if (
+      // Make sure we haven't already added this applicant:
+      !modifiedFormData.applicants.some(
+        a =>
+          a.applicantName.first === name.first ||
+          a.applicantEmailAddress === email,
+      )
+    ) {
+      modifiedFormData.applicants = [
+        newApplicant,
+        ...modifiedFormData.applicants,
+      ];
+    }
   } else {
-    nameKey = 'certifierName';
+    // No applicants yet. Create array and add ours:
+    modifiedFormData.applicants = [newApplicant];
   }
-
-  return nameKey;
+  return modifiedFormData;
 }
 
-// Extracting this to a function so there aren't a thousand identical
-// ternaries we have to change later
-export function sponsorWording(formData, isPosessive = true, cap = true) {
-  let retVal = '';
-  if (formData?.certifierRole === 'sponsor') {
-    retVal = isPosessive ? 'your' : 'you';
-  } else {
-    retVal = isPosessive ? 'sponsor’s' : 'sponsor';
-  }
```

Comment:
Removed since we no longer have the information on whether or not the user is also the sponsor.